commands/core/cli.drush.inc

```diff
@@ -70,14 +70,98 @@ function drush_cli_core_cli() {
 function _drush_core_cli_get_commands() {
   $commands = drush_get_commands();
   $ignored_commands = ['help', 'drush-psysh', 'php-eval', 'core-cli', 'php'];
+  $php_keywords = _drush_core_cli_get_php_keywords();
 
   foreach ($commands as $name => $config) {
-    // Ignore some commands that don't make sense inside PsySH, are hidden, or
-    // are aliases.
-    if (in_array($name, $ignored_commands) || !empty($config['hidden']) || ($name !== $config['command'])) {
+    // Ignore some commands that don't make sense inside PsySH, are PHP keywords
+    // are hidden, or are aliases.
+    if (in_array($name, $ignored_commands) || in_array($name, $php_keywords) || !empty($config['hidden']) || ($name !== $config['command'])) {
       unset($commands[$name]);
     }
+    else {
+      // Make sure the command aliases don't contain any PHP keywords.
+      if (!empty($config['aliases'])) {
+        $commands[$name]['aliases'] = array_diff($commands[$name]['aliases'], $php_keywords);
```

Comment:
Yes, we could try to get this upstream. Let's keep it here for now though, as we can have this working before the 8.0.0 release, if we wait for upstream, we will likely not.